src/libcollections/vec.rs

```diff
@@ -456,8 +458,9 @@ impl<T> Vec<T> {
     }
 
     /// Reserves the minimum capacity for exactly `additional` more elements to
-    /// be inserted in the given `Vec<T>`. Does nothing if the capacity is already
-    /// sufficient.
+    /// be inserted in the given `Vec<T>`. After calling `reserve_exact`,
+    /// capacity will be greater than or equal to `self.len() + additional`.
```

Comment:
I don't think that's correct though? If the capacity already exceeds `self.len() + additional` then this method will not shrink the vector.

If the capacity is *less* than `self.len() + additional` then this will return with capacity equal to that, but as-written this documentation is correct. After this method you cannot rely on the capacity being exactly equal to the request, only greater than or equal to the requested total capacity.